.github/workflows/release.yaml

```diff
@@ -76,31 +76,23 @@ jobs:
           toolchain: ${{ matrix.rust }}
           args: --release --target ${{ matrix.job.target }}
 
-      - name: install strip command
-        shell: bash
-        run: |
-          if [[ ${{ matrix.job.target }} == aarch64-unknown-linux-gnu ]]; then
-              sudo apt update
-              sudo apt-get install -y binutils-aarch64-linux-gnu
-          fi
+      - name: Generate completions
+        uses: actions-rs/cargo@v1
+        with:
+          command: run
+          use-cross: ${{ matrix.job.use-cross }}
+          toolchain: ${{ matrix.rust }}
+          args: --release --features=completions --target ${{ matrix.job.target }}
+
       - name: Packaging final binary
         if: ${{ !contains(github.ref_name, '/') }}
         shell: bash
         run: |
           cd target/${{ matrix.job.target }}/release
 
-          ####### reduce binary size by removing debug symbols #######
-          BINARY_NAME=rustic${{ matrix.job.binary-postfix }}
-          if [[ ${{ matrix.job.target }} == aarch64-unknown-linux-gnu ]]; then
-            GCC_PREFIX="aarch64-linux-gnu-"
-          else
-            GCC_PREFIX=""
-          fi
-          "$GCC_PREFIX"strip $BINARY_NAME
-
           ########## create tar.gz ##########
           RELEASE_NAME=rustic-${GITHUB_REF_NAME}-${{ matrix.job.target}}
-          tar czvf $RELEASE_NAME.tar.gz $BINARY_NAME
+          tar czvf $RELEASE_NAME.tar.gz rustic${{ matrix.job.binary-postfix }} -C ../../../target ./completions
```

Comment:
I'm a bit unsure if the completions should be released withing the tar.gz containing the binary. I would tend to creating a `completions.tar.gz` and release this as extra file.